sql/core/src/main/scala/org/apache/spark/sql/execution/command/tables.scala

```diff
@@ -720,8 +721,8 @@ case class DescribeColumnCommand(
     }
 
     val catalogTable = catalog.getTempViewOrPermanentTableMetadata(table)
-    val colStats = catalogTable.stats.map(_.colStats).getOrElse(Map.empty)
-    val cs = colStats.get(field.name)
+    val colStats = getColStats(catalogTable)
+    val cs = colStats.get(getColumnName(field.name))
 
```

Comment:
How about just simply writing it like this?
```

    val catalogStats = catalog.getTempViewOrPermanentTableMetadata(table).stats
    val cs = if (conf.caseSensitiveAnalysis) {
      catalogStats.flatMap { cs => cs.colStats.get(field.name) }
    } else {
      catalogStats.flatMap { cs =>
        cs.colStats.map { case (k, v) => (k.toLowerCase(Locale.ROOT), v) }
          .get(field.name.toLowerCase(Locale.ROOT))
      }
    }
```

Reply:
I have reworked on the change suggested. Could you review?